src/Microsoft.AspNet.Mvc.Razor/Compilation/RoslynCompilationService.cs

```diff
@@ -21,20 +21,18 @@
 using Microsoft.Dnx.Compilation.CSharp;
 using Microsoft.Extensions.PlatformAbstractions;
 using Microsoft.Extensions.Options;
-using Microsoft.AspNet.Diagnostics;
 
 namespace Microsoft.AspNet.Mvc.Razor.Compilation
 {
     /// <summary>
     /// A type that uses Roslyn to compile C# content.
     /// </summary>
-    public class RoslynCompilationService : ICompilationService
+    public abstract class RoslynCompilationService : ICompilationService
```

Comment:
If one of the derived types is meant to be throw-away code, I'd rather we duplicate this so we don't need to deal with this when we eventually remove the DNX support.